naming/src/main/resources/org/jboss/as/naming/subsystem/LocalDescriptions.properties

```diff
@@ -23,6 +23,16 @@ binding.rebind.module=The module to load the object factory from for object fact
 binding.rebind.class=The object factory class name for object factory bindings
 binding.rebind.lookup=The entry to lookup in JNDI for lookup bindings
 binding.rebind.environment=The environment to use on object factory instance retrieval
+binding.property= The property stores the key-value of properties binding
```

Comment:
* _binding.binding-type_ above needs to be updated to include _properties_ in the description